source/includes/library-reference/_brain.html.md

```diff
@@ -0,0 +1,47 @@
+# Brain Class
+
+Used to manage a BRAIN instance and talk with the server backend. Contains
+state information about the BRAIN and can be used to upload/download inkling
+to and from the BRAIN on the server.
+
+Requires a shared pointer to a `bonsai::Config` and a BRAIN name.
```

Comment:
Python has no shared pointers and the scope operator is `.` not `::`

Reply:
Sure, but this is documenting more than just python. Could change to `Requires a `Config` and a BRAIN name.`